MapboxNavigation/RouteMapViewController.swift

```diff
@@ -488,7 +489,7 @@ class RouteMapViewController: UIViewController {
         var margin: CGFloat = 0.0
         if #available(iOS 11.0, *) { margin = view.safeAreaInsets.bottom }
         let containerHeight = endOfRouteContainerView.frame.height - margin
-        let bottom = self.endOfRouteShowConstraint.isActive ? containerHeight : bottomBannerView.bounds.height
+        let bottom = self.endOfRouteShowConstraint.isActive ? containerHeight : bottomBannerView?.bounds.height ?? 40
```

Comment:
@mapbox/navigation-ios Is there anything else we can do here other than falling back to 40?

Reply:
This should be factored out to a UIPercentDrivenInteractiveTransition or another interactive transitioning class but I think this works for now. /cc @JThramer

Reply:
@JThramer @frederoni cool, mind giving this a 🍏  if everything else is good?